src/DI/OrmExtension.php

```diff
@@ -53,11 +55,18 @@ public function loadConfiguration(): void
 	public function loadDoctrineConfiguration(): void
 	{
 		$builder = $this->getContainerBuilder();
+
 		$config = $this->validateConfig($this->defaults['configuration'], $this->config['configuration']);
 		$config = Helpers::expand($config, $builder->parameters);
 
+		$configurationClass = $this->config['configurationClass'];
+
+		if ($configurationClass !== Configuration::class && !is_subclass_of($configurationClass, Configuration::class)) {
```

Comment:
Have you considered `is_a` (http://php.net/manual/en/function.is-a.php)? 

```php
if (!is_a($configurationClass, Configuration::class)) {
// throw
}
```

Reply:
tnx for hint